### ISSUE TITLE
[Accessibility] A11y_.Net Editor_Screen Reader: Narrator/NVDA is not mentioning anything about the dialog box after pressing Developer : inspect tokens #107

### ISSUE THREAD

**Issue by LadyNaggaga** [Microsoft/monaco-editor#1165]
.NET Online REPL  is having the following accessibility issue.  Please see this [issue](https://github.com/dotnet/try/issues/107) . I have also included the details below 

> **Environment Details**
> https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
> **Browser:** Edge (41.16299.402.0)
> Chrome (67.0.3396.87)
> **Screen Reader:** Narrator
> NVDA
> 
> **Additional Details:**
> MAS Violated: MAS1.3.2
> 
> **Repro steps:**
> 
> 1. Hit the URL https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
> 2. Tab till “Enter focus mode” and press enter
> 3. Type any code and press F1 and tab till Developer : inspect tokens and press enter
> 4. Verify Whether the Screen Reader is mentioning if the dialog box got opened.
> 
> **Actual Result:**
> Screen Reader is not mentioning anything about the dialog box .
> 
> **Expected Result:**
> Screen Reader should mention that the inspect token dialog box has been opened.
> 
> **User Impact:**
> Screen reader users find difficult in understanding the tokens in the code if it did not mention about the dialog box.
> 
> **MAS Reference:**
> [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B1ee227a7-e3c5-47b8-ab18-48ca8595077b%7D)
> 
> [screen reader is not mentioning anything about the dialog box after pressing Developer inspect tokens.pptx](https://github.com/dotnet/try/files/2188385/screen.reader.is.not.mentioning.anything.about.the.dialog.box.after.pressing.Developer.inspect.tokens.pptx)

**Comment by alexandrudima** [comment]
This is a developing time command that is destined for folks writing grammars for the Monaco Editor.

In other words, this command does nothing useful for end-users and is not necessary at all to invoke it whatsoever. This command is useful for Developers of the Monaco Editor and for Developers of colorizer grammars.
